components/lv-ts-tableprovider/LVSimple/pom.xml

```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+    <!-- vim: set tabstop=4 softtabstop=0 expandtab shiftwidth=4 smarttab : -->
+
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>com.example</groupId>
+    <artifactId>LiveViewSimple</artifactId>
+    <packaging>ep-liveview-fragment</packaging>
+    <version>0.0.1-SNAPSHOT</version>
+    
+    <name>LiveView sample that uses TS TableProviders</name>
+    <description>LiveView TableProvider that allows LiveViewClients to get system logs, snapshots, system table contents, etc.</description>
+
+    <!-- common definitions for this version of StreamBase -->
+    <parent>
+        <groupId>com.tibco.ep.sb.parent</groupId>
+        <artifactId>ep-liveview-fragment</artifactId>
+        <version>10.5.1-SNAPSHOT</version>

```

Comment:
missing a <relativePath/> inide the parent

Reply:
Sorry, I haven't done (correctly anyway!) a project with modules. The sub project should look like this?

    <parent>
        <groupId>com.tibco.ep.sb.parent</groupId>
        <artifactId>ep-liveview-fragment</artifactId>
        <version>10.5.1-SNAPSHOT</version>
        <relativePath>../pom</relativePath>
    </parent>